src/Lykke.Service.PayInternal.AzureRepositories/PaymentRequest/PaymentRequestRepository.cs

```diff
@@ -34,6 +38,55 @@ IEnumerable<PaymentRequestEntity> paymentRequestEntities
             return Mapper.Map<IEnumerable<Core.Domain.PaymentRequests.PaymentRequest>>(paymentRequestEntities).ToList();
         }
 
+        public async Task<bool> HasAnyPaymentRequestAsync(string merchantId)
+        {
+            var entity = await _storage.GetTopRecordAsync(PaymentRequestEntity.ByMerchant.GeneratePartitionKey(merchantId));
+            return entity != null;
+        }
+
+        public async Task<IReadOnlyList<IPaymentRequest>> GetByFilterAsync(PaymentsFilter paymentsFilter)
+        {
+            var filter = nameof(Entity.PartitionKey)
+                .PropertyEqual(PaymentRequestEntity.ByMerchant.GeneratePartitionKey(paymentsFilter.MerchantId))
+                .And(nameof(Entity.Initiator).PropertyEqual(LykkePayConstants.ApiPaymentRequestInitiator));
```

Comment:
No need to have an instance, you can use nameof(PaymentRequestEntity.Initiator), argument expression is never evaluated in nameof.